### ISSUE TITLE
What is the recomended way of supporting limited amount of cards?

### ISSUE THREAD

**Issue by stefek99** [driftyco/ionic-ion-tinder-cards#86]
![bug-report](https://cloud.githubusercontent.com/assets/249703/14569282/a1452470-0335-11e6-9851-a2bd627075a5.jpg)

See also #85 

If we have unlimited cards - no problem.

As we have finite amount - accessing third form the end triggers an error (see image attached)

**Comment by dan-diddy** [comment]
On line 385 in your screenshot you check whether there's a second card. Why not just use that to check for the third card? Presumably you could put that check in a while loop. Alternatively, you also know the length of the array so you could easily write a for loop.

**Comment by stefek99** [comment]
> On line 385 in your screenshot you check whether there's a second card. Why not just use that to check for the third card? Presumably you could put that check in a while loop. Alternatively, you also know the length of the array so you could easily write a for loop.

@dan-diddy Yep, checking if the array contains the element would fix my issue in my copy of the library, that would have to be served from my local server instead of the CDN.

How about all the other libraries in the wild?

My gut feeling would be to fix the library.

**Comment by keithdmoore** [comment]
@stefek99 If you are willing to fix it, I am willing to merge the PR

**Comment by stefek99** [comment]
As per comment here: https://github.com/driftyco/ionic-ion-tinder-cards/issues/81#issuecomment-172984605

_(will give a go to a fork first)_
